src/Components/Navigation/Footer.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import './Footer.css';
+import { NavLink } from 'react-router-dom';
+import Logo from '../../Assets/Images/TCK PNG Logo.png';
+import facebookLogo from '../../Assets/Images/facebook-footer-logo.svg';
```

Comment:
make these logo names uppercase to be consistent with the other imports
```suggestion
import FacebookLogo from '../../Assets/Images/facebook-footer-logo.svg';
```